platform/jvm/capture/build.gradle.kts

```diff
@@ -113,7 +113,7 @@ cargoNdk {
     targets = arrayListOf("arm64")
     // enable 16 KB ELF alignment on Android to support API 35+
     extraCargoEnv = mapOf(
-      "RUSTFLAGS" to "-C link-args=-Wl,-z,max-page-size=16384,--build-id",
+      "RUSTFLAGS" to "-C codegen-units=1 -C embed-bitcode -C lto=fat -C opt-level=z -C link-args=-Wl,-z,max-page-size=16384,--build-id",
```

Comment:
is this still required?

Reply:
This is just for the cargo build so I figured I'd leave this in so that there is parity between all cargo and bazel. All the other stuff just targets the bazel pipeline so this is still necessary for the gradle build